src/main/java/com/khutircraftubackend/auth/AuthenticationController.java

```diff
@@ -67,7 +67,7 @@ public ResponseEntity<?> register(@Valid @RequestBody RegisterRequest registerRe
     public ResponseEntity<String> confirmUser(@RequestBody ConfirmationRequest confirmationRequest) {
         try {
             authenticationService.confirmUser(confirmationRequest.email(), confirmationRequest.jwt());
-            return ResponseEntity.ok("User confirmed successfully.");
+            return ResponseEntity.ok("UserEntity confirmed successfully.");
```

Comment:
`User  --> UserEntity` looks like "wildcard replacement" all over the place
did we really want that?